pkg/internal/cnsoperator/cnsfilevolumeclient/v1alpha1/groupversion_info.go

```diff
@@ -1,39 +0,0 @@
-/*
-Copyright 2020 The Kubernetes authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Package v1alpha1 contains API Schema definitions for the cns v1alpha1 API group
-// +kubebuilder:object:generate=true
-// +groupName=cns.vmware.com
-package v1alpha1
```

Comment:
Can you clarify what is the change in this file? GitHub shows this file as deleted.

Reply:
I am renaming the `internal` folder name to `internalapis`. 


Having internal in the folder structure has a special convention in golang

```
internal/ is a special directory name recognised by the go tool which will prevent one package from being imported by another unless both share a common ancestor.
```
When we `pkg/internal` we will not be able to refer the any of the packages from outside the folder unless they share common ancestor. So, using `internalapis`.